src/app/shared/navbar/searchbar/searchbar.ts

```diff
@@ -0,0 +1,51 @@
+import {Component, HostBinding} from '@angular/core';
+import {Router} from '@angular/router';
+import {FormControl} from '@angular/forms';
+import {MdSnackBar} from '@angular/material';
+import {Observable} from 'rxjs';
+
+import {DocumentationItems, DocItem} from '../../documentation-items/documentation-items';
+
+
+@Component({
+  selector: 'search-bar-component',
+  templateUrl: './searchbar.html',
+  styleUrls: ['./searchbar.scss']
+})
+
+export class SearchBar {
+  @HostBinding('class.expanded') _isExpanded = false;
```

Comment:
Move this to the host object in the Component metadata? Generally prefer that instead of using the decorator.